Sources/idt/idt.cc

```diff
@@ -43,10 +44,49 @@ inplace("inplace", llvm::cl::init(false),
         llvm::cl::desc("Apply suggested changes in-place"),
         llvm::cl::cat(idt::category));
 
+llvm::cl::list<std::string>
+ignored_functions("ignore",
+                  llvm::cl::desc("Ignore one or more functions"),
+                  llvm::cl::value_desc("function-name[,function-name...]"),
+                  llvm::cl::cat(idt::category));
+
+llvm::cl::opt<bool>
+allow_known_builtins("allow-known-builtins", llvm::cl::init(false),
+                     llvm::cl::desc("Do not automatically ignore known built-in functions"),
+                     llvm::cl::cat(idt::category));
+
 template <typename Key, typename Compare, typename Allocator>
 bool contains(const std::set<Key, Compare, Allocator>& set, const Key& key) {
   return set.find(key) != set.end();
 }
+
+const std::set<std::string>& get_ignored_functions() {
+  static auto kIgnoredFunctions = [&]() -> std::set<std::string> {
+      if (ignored_functions.empty()) {
+        if (!allow_known_builtins)
+          return kDefaultIgnoredFunctions;
+
+        return {};
+      }
+
+      std::set<std::string> user_ignored_functions;
+      if (!allow_known_builtins)
+        user_ignored_functions.insert(
+            kDefaultIgnoredFunctions.begin(), kDefaultIgnoredFunctions.end());
+
+      for (const std::string& func_list : ignored_functions) {
+        // Allow function names to be comma-separated without specifying the 
+        // `llvm::cl::CommaSeparated` modifier.
```

Comment:
Is there a benefit or some reasoning to not use the `llvm::cl::CommaSeparated`?

Reply:
At one point, if you used `llvm::cl::CommaSeparated`, you couldn't put the argument on the command line more than once. This was a _long_ time ago (like... LLVM 3.9, I think) and, admittedly, I've done it out of habit ever since then. I haven't done any testing yet to see if it works both ways. If it does, I'm much more in favor of adding the modifier instead.

Reply:
Confirmed that whatever limitations I had back in 2017 are no longer a problem `-ignore foo -ignore bar` works just as well as `-ignore foo,bar` with `llvm::cl::CommaSeparated`.